### DISCUSSION TITLE
Find button

### DISCUSSION THREAD

**Discussion by NN---** [community/community#2828] [Mobile]
Is there find in file button in the mobile application ?

**Comment by candyho** [comment]
We have a "jump to file" button when you're reviewing a PR - go to PR view > tap "Review" > tap the file icon at the bottom right corner > select the file you'd like to review. Is this what you're looking for? Hope this helps!

**Comment by NN---** [comment]
I need text search in the files, in PRs, not file search.

**Comment by cdce8p** [comment]
That's also something I would use a lot. On PC you can just do `Crtl+F` most of the time and most mobile browsers support some kind of `Find on page` feature. However those are not available in the App.

A better title for this discussion might be: `Search in current file` :)

**Comment by danihodovic** [comment]
CTRL+F would be useful for navigating code.

**Reply by NN---** [reply]
> 
> CTRL+F would be useful for navigating code.

We are talking about mobile application :)

**Reply by danihodovic** [reply]
So am I :)

**Comment by virtualdj** [comment]
Agreed, a basic **find** feature inside the code viewer should be implemented to navigate big files easily.

**Comment by PranamBhat** [comment]
Click on "Search or jump to ..." Enter your file/folder name. That's all.

**Comment by virtualdj** [comment]
> Click on "Search or jump to ..." Enter your file/folder name. That's all.

The OP (and me) is asking about searching in the current file, not a file/folder search in the repository.
Try to search for example the word _var_ into a Javascript source file using the mobile application... you can't!

**Comment by PranamBhat** [comment]
Hello!

This is possible in **Mobile Application**. There are few restrictions from **GitHub** as well:

1) You must be signed into a user account on GitHub to search for code across all public repositories.
2) Code in forks is only searchable if the fork has more stars than the parent repository. Forks with fewer stars than the parent repository are not indexed for code search. To include forks with more stars than their parent in the search results, you will need to add fork:true or fork:only to your query. For more information, see "Searching in forks."
3) Only the default branch is indexed for code search.
4) Only files smaller than 384 KB are searchable.
5) Only repositories with fewer than 500,000 files are searchable.
6) Only repositories that have had activity or have been returned in search results in the last year are searchable.
7) Except with filename searches, you must always include at least one search term when searching source code. For example, searching for language:javascript is not valid, while amazing language:javascript is.
8) At most, search results can show two fragments from the same file, but there may be more results within the file.
9) You can't use the following wildcard characters as part of your search query: . , : ; / \ ` ' " = * ! ? # $ & + ^ | ~ < > ( ) { } [ ]. The search will simply ignore these symbols.


For more information, check out here : [https://docs.github.com/en/github/searching-for-information-on-github/searching-code](https://docs.github.com/en/github/searching-for-information-on-github/searching-code)

**Reply by virtualdj** [reply]
This may act as a *partial workaround* (actually it is NOT... too many limitations!) but anyway it's not what the OP is asking. To have a full functional code editor on the mobile application, a search feature is necessary on the editor itself. Even Notepad has it!
